applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_axisymmetric_integration_coefficients.cpp

```diff
@@ -44,7 +41,8 @@ KRATOS_TEST_CASE_IN_SUITE(AxisymmetricIntegrationCoefficientsCalculator_ReturnsC
     // Assert
     // The expected number is calculated as follows:
     // 2.0 * pi * 0.8 (radius) * 2.0 (detJ) * 0.5 (weight) = 5.02655
-    KRATOS_EXPECT_NEAR(calculated_coefficients[0], 5.02655, 1e-5);
+    KRATOS_EXPECT_EQ(calculated_coefficients.size(), 1);
+    KRATOS_EXPECT_NEAR(calculated_coefficients[0], 5.026548245743669, Defaults::absolute_tolerance);
```

Comment:
It seems like the calculated integration coefficient deviates more than the default absolute tolerance from the expected value. To me, that is a bit unexpected. Instead of modifying the expected value, I would suggest to use a relative tolerance. I have checked that it works for this case.
```suggestion
    KRATOS_EXPECT_RELATIVE_NEAR(calculated_coefficients[0], 5.02655, Defaults::relative_tolerance);
```

Also for the other cases where you have tweaked the expected value, I would suggest to go back to the original expected value and assert using a relative tolerance.

Reply:
changed